### PR TITLE
Abstracted common requirements  for workloads

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: tasks/run-ansible-with-podman.xml [hunk 1]
```diff
@@ -15,29 +15,15 @@
...
-        </para>
-      </listitem>
-      <listitem>
+  <info>
+    <title>Running the &ansible; workload using &podman;</title>
+    <abstract>
+      <para>
+        &ansible; is a suite of tools for managing and provisioning data
+        centers via definition files. This article describes how to run
+        &ansible; on the &productname; (&productnameshort;).
```
Comment (dariavladykina) [inline tasks/run-ansible-with-podman.xml:24]:
```suggestion
        &ansible; on &productname; (&productnameshort;).
```

File: tasks/run-firewalld-with-podman.xml [hunk 2]
```diff
@@ -15,44 +15,25 @@
...
-    <important>
+  <info>
+    <title>Running &firewalld; using &podman;</title>
+    <abstract>
       <para>
-        The &firewalld; container needs access to the host network and needs to
-        run as a privileged container. The container image uses the system dbus
-        instance. Therefore, you need to install <package>dbus</package> and
-        <package>polkit</package> configuration files first.
+        This article describes how to run a containerized &firewalld; on the
```
Comment (dariavladykina) [inline tasks/run-firewalld-with-podman.xml:22]:
```suggestion
        This article describes how to run a containerized &firewalld; on
```

File: tasks/run-grafana-with-podman.xml [hunk 3]
```diff
@@ -15,32 +15,18 @@
...
-        </para>
-      </listitem>
-    </itemizedlist>
-  </section>
+  <info>
+    <title>Running the &grafana; workload using &podman;</title>
+    <abstract>
+      <para>
+        This article describes how to run the &grafana; visualization tool on
+        the &productname; (&productnameshort;). This workload adds a Web-based
```
Comment (dariavladykina) [inline tasks/run-grafana-with-podman.xml:23]:
```suggestion
        &productname; (&productnameshort;). This workload adds a Web-based
```

**Review by dariavladykina — COMMENTED** [review]
LGTM, just no article for product names. Thanks!

**Review by tbazant — COMMENTED** [review]
suggestions included

**Pull request merged**